### PR TITLE
Remove special HTTP HEAD processing

### PR DESCRIPTION
**Pull request by SenorSmartyPants**
Removing this allows HTTP 206 Partial Content responses and lets some clients(popcorn hour namely) play static videos from JF.

### CODE REVIEW

File: Jellyfin.Api/Helpers/FileStreamResponseHelpers.cs [hunk 1]
```diff
@@ -70,12 +64,6 @@ public static class FileStreamResponseHelpers
         {
             httpContext.Response.ContentType = contentType;
```
Comment (crobibero) [inline Jellyfin.Api/Helpers/FileStreamResponseHelpers.cs:65]: this line can also be removed

**Review by crobibero — COMMENTED** [review]
Please also remove the unused parameters from the functions

File: Jellyfin.Api/Helpers/FileStreamResponseHelpers.cs (hunk 1 above)
Comment (SenorSmartyPants) [reply Jellyfin.Api/Helpers/FileStreamResponseHelpers.cs]: Updates committed

**Comment by crobibero** [comment]
Review: ensure DLNA headers are still returned

File: Jellyfin.Api/Helpers/FileStreamResponseHelpers.cs [hunk 2]
```diff
@@ -59,23 +51,13 @@ public static class FileStreamResponseHelpers
...
         /// <param name="path">The path to the file.</param>
         /// <param name="contentType">The content type of the file.</param>
-        /// <param name="isHeadRequest">Whether the current request is a HTTP HEAD request so only the headers get returned.</param>
         /// <param name="httpContext">The current http context.</param>
         /// <returns>An <see cref="ActionResult"/> the file.</returns>
         public static ActionResult GetStaticFileResult(
             string path,
             string contentType,
-            bool isHeadRequest,
             HttpContext httpContext)
```
Comment (crobibero) [inline Jellyfin.Api/Helpers/FileStreamResponseHelpers.cs:59]: httpContext is no longer used, should be removed from the function

Comment (SenorSmartyPants) [reply Jellyfin.Api/Helpers/FileStreamResponseHelpers.cs]: Removed. VS Code isn't highlighting unused parameters for me. So that's weird.

**Review by crobibero — APPROVED** [review]

**Comment by SenorSmartyPants** [comment]
Accidentally pushed another change to my main, instead of forking. Then I force pushed to remove it.

**Review by 1337joe — APPROVED** [review]

**Review by Bond-009 — APPROVED** [review]

**Pull request merged**